src/js/components/DataTable/doc.js

```diff
@@ -122,6 +122,7 @@ export const doc = DataTable => {
           ]),
           PropTypes.string,
         ]),
+        units: PropTypes.string,
```

Comment:
This is actually an object that accepts all props of Text component

Reply:
This one is actually the string units rather than props. For example, "(TiB)" or "%" etc.